chainer_chemistry/dataset/preprocessors/ggnn_preprocessor.py

```diff
@@ -45,4 +51,45 @@ def get_input_features(self, mol):
         type_check_num_atoms(mol, self.max_atoms)
         atom_array = construct_atomic_number_array(mol, out_size=self.out_size)
         adj_array = construct_discrete_edge_matrix(mol, out_size=self.out_size)
+
         return atom_array, adj_array
+
+    def create_dataset(self, *args, **kwargs):
```

Comment:
is it possible to return NumpyTupleDataset for padding pattern dataset for now?

Reply:
fixed!